src/woo_publications/metadata/management/commands/update_information_categories_from_waardenlijst.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+from woo_publications.metadata.update_informatie_category import (
+    InformatieCategoryWaardenlijstError,
+    update_informatie_category,
```

Comment:
```suggestion
    update_information_category,
```
weird NL/EN mix in the code, let's try to keep code as much English as possible